blocks/calendar-events/calendar-events.css

```diff
@@ -0,0 +1,130 @@
+.calendar-events.block {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: auto;
+    justify-items: center;
+    gap: 28px;
+    margin: 0 auto;
+}
+
+.calendar-events.block .calendar-event {
+    border: solid 1px transparent;
+    border-radius: 10px;
+    box-shadow: 0 4px 4px var(--gray-box-shadow);
+    width: 100%;
+}
+
+.calendar-events.block .calendar-event > .event-card {
+    padding: 20px 26px;
+
+}
+
+.calendar-events.block .calendar-event > .event-card p:first-of-type {
+    color: var(--gray);
+    font-size: var(--body-font-size-s);
+    text-transform: uppercase;
+    line-height: 14px;
+    margin: 0;
+}
+
+.calendar-events.block .calendar-event > .event-card p:nth-of-type(2) {
+    color: var(--black);
+    font-size: var(--body-font-size-m);
+    font-weight: var(--font-weight-bold);
+    line-height: 22px;
+    margin-bottom: 10px;
+}
+
+.calendar-events.block .calendar-event > .event-card > div:first-of-type {
+    margin-bottom: 10px;
+}
+
+.calendar-events.block .calendar-event > .event-card > div:nth-of-type(2) {
+    color: var(--default-text-color);
+    font-family: var(--body-font-family);
+    font-size: var(--body-font-size-xs);
+    line-height: 17px;
+}
+
+.calendar-events.block .calendar-event > .date-card {
+    display: grid;
+    grid-template-columns: 80px auto;
+    padding: 20px;
+}
+
+.calendar-events.block .calendar-event > .date-card  > div:first-of-type {
+    background: var(--red);
+    height: 64px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin: 0 14px 0 0;
+    border: solid 1px transparent;
+}
+
+.calendar-events.block .calendar-event > .date-card  > div:first-of-type > h3,
+.calendar-events.block .calendar-event > .date-card  > div:first-of-type > p {
+    color: var(--white);
+    font-weight: var(--font-weight-bold);
+    line-height: 22px;
+    margin: 0;
+}
+
+.calendar-events.block .calendar-event > .date-card  > div:first-of-type > h3 {
+    font-size: var(--body-font-size-xxl);
+}
+
+.calendar-events.block .calendar-event > .date-card  > div:first-of-type > p {
+    font-size: var(--body-font-size-s);
+    text-transform: uppercase;
+}
+
+.calendar-events.block .calendar-event > .date-card  > div:nth-of-type(2) {
+    font-family: var(--body-font-family);
+    font-size: var(--body-font-size-m);
+    line-height: 22px;
+}
+
+.calendar-events.block .calendar-event > .date-card  > div:nth-of-type(2) > p {
+    font-size: var(--body-font-size-l);
+}
+
+@media screen and (max-width: 300px){
+    .calendar-events.block {
+        width: 340px;
+    }
+}
+
+@media screen and (min-width: 600px){
+    .calendar-events.block {
+        width: 550px;
+    }
+}
+
+@media screen and (min-width: 900px) and (max-width: 1199px){
```

Comment:
@bstopp this is the odd behavior between these two pixel widths.  These essentially make everything larger, but just in this section.  Should I expand the breakpoint definitions and remove the `max-width`?  That's not a problem.

Reply:
I see. I think the only way to resolve this will be to add additional context at the larger break points when this block is in a layout section.

Let's eliminate the max with qualifier - and when we get the final layout logic & styling, we'll adjust font size back down at 1200px. 

make sense?